### PR TITLE
Create .gitattributes file to avoid installing test folder

### PR DESCRIPTION
**Pull request by VincentLanglet**
Hi @ezimuel,

I noticed when installing this lib that the test folder is downloaded too.
This PR fix this.

### CODE REVIEW

**Comment by VincentLanglet** [comment]
Friendly ping @ezimuel, if you have time :)

**Comment by VincentLanglet** [comment]
Friendly ping @ezimuel, if you have time :)

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.github export-ignore
+tests export-ignore
+phpunit.xml.dist export-ignore
```
Comment (reedy) [inline .gitattributes:3]: You should include `.gitignore` and `.gitattributes` itself here too :)

**Review by reedy — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 1 above)
Comment (VincentLanglet) [reply .gitattributes:3]: done

**Comment by VincentLanglet** [comment]
Friendly ping @ezimuel, if you have time :)

**Comment by VincentLanglet** [comment]
Friendly ping @ezimuel, if you have time to review this :)

Thanks

**Review by reedy — APPROVED** [review]

**Comment by ezimuel** [comment]
Thanks @VincentLanglet and @reedy

**Comment by ezimuel** [comment]
Released in [4.0.0](https://github.com/ezimuel/guzzlestreams/releases/tag/4.0.0).

**Pull request merged**